pkg/util/kube/utils.go

```diff
@@ -161,8 +177,8 @@ func IsCRDReady(crd *apiextv1beta1.CustomResourceDefinition) bool {
 }
 
 // IsUnstructuredCRDReady checks an unstructured CRD to see if it's ready, with both the Established and NamesAccepted conditions.
-// TODO: Delete this function and use IsCRDReady when the upstream runtime.FromUnstructured function properly handles int64 field conversions.
-// Duplicated function because the velero install package uses IsCRDReady with the beta types.
+// TODO: Delete this function and use Isv1beta1CRDReady when the upstream runtime.FromUnstructured function properly handles int64 field conversions.
```

Comment:
The restore code which uses IsUnstructuredCRDReady doesn't really need to deal with v1/v1beta1 issues right now. I'm guessing that by the time this code is refectored (if it is ever refactored) we'll no longer be supporting v1beta1, so the above (and below) lines should probably reference `IsV1CRDReady` instead.

Reply:
Ah, that was a find/replace gone wrong :) I'll fix it.

Reply:
I checked yesterday, and the upstream fix is available in the v0.20.0 version of the API libraries. Once we upgrade those, a unit test which checks for the upstream error will fail which might encourage the refactoring at that stage.

Reply:
Ahh. OK. If we choose to do that soon we'll need to deal with v1 vs v1beta1 issues in the refactor.

Reply:
I think that's fine. The issue was that the typed checks were failing with CRDs which included int64 fields. Velero CRDs don't use those, so the installer wasn't hitting the problem, but restoring backups which included CRDs with these fields was failing. The workaround was to use the unstructured fields. I don't know whether it really matters enough at this point to go back and refactor the restore CRD ready checks to use the v1/v1beta1 functions, but if we do we just need to make sure that the int64 issue is gone by testing backup/restore with a CRD which uses this sort of field.